src/ceph-disk

```diff
@@ -2191,11 +2353,65 @@ def mount_activate(
 
     path = mount(dev=dev, fstype=fstype, options=mount_options)
 
+    # check if the disk is deactive, change the journal owner, group
+    # mode for correct user and group.
+    if os.path.exists(os.path.join(path, 'deactive')):
+        # logging to syslog will help us easy to know udev triggered failure
+        if not reactivate:
+            unmount(path)
```

Comment:
is there a reason to use **syslog.syslog** here instead of **LOG.info** as it is done elsewhere ?